cmake/SetupCodeMetrics.cmake

```diff
@@ -49,7 +49,28 @@
 # Note: Only supported for gnu or clang.
 ########################################
 if (ENABLE_COVERAGE)
-    include(blt/cmake/CodeCoverage.cmake)
+    ##########################################################################
+    # Setup coverage compiler flags 
+    ##########################################################################
+    # Set the actual flags for coverage in the COVERAGE_FLAGS variable 
+    # Note: '--coverage' is equivalent to '-fprofile-arcs -ftest-coverage' for compiling and '-lgcov' for linking
+    # Additional flags that might be useful: 
+    #       " -fno-inline -fno-inline-small-functions -fno-default-inline"
+    blt_append_custom_compiler_flag(FLAGS_VAR   COVERAGE_FLAGS 
+                                    DEFAULT     "--coverage")
```

Comment:
Is ``--coverage`` valid on all compilers?  Or only on gcc / clang ?
The previous version had a check that ensured that coverage was only supported on gcc. 
Otherwise, you'll need to set flags for the other compilers.